src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -451,6 +451,11 @@ case class IndexLogEntry(
     relations.head.data.properties.content.fileInfos
   }
 
+  @JsonIgnore
+  lazy val sourceFilesBytes: Long = {
```

Comment:
`sourceFilesTotalSizeInBytes` or `sourceFilesSizeInBytes`. If you just end with `Bytes`, it sounds like you are referring to the actual bytes. Applies to other places if applicable.